golem_messages/message.py

```diff
@@ -454,7 +454,7 @@ def __init__(self, tasks=None, **kwargs):
         :param list tasks: list of tasks information (subset of
                            taskserver.get_tasks_headers())
         """
-        self.tasks = tasks or []
+        kwargs['tasks'] = tasks or []

```

Comment:
Is there any reason that this one is different from the others?

Reply:
No reason, just lost in development :). Fixed